src/main/java/com/getindata/kafka/connect/iceberg/sink/IcebergChangeEvent.java

```diff
@@ -410,6 +413,12 @@ private List<Types.NestedField> icebergSchema(JsonNode eventSchema, String schem
             columnId += subSchema.size();
             break;
           default: //primitive types
+            if (fieldName.equals(partitionColumn)) {
+              // if it is the partition column, swap its type to timestamp
+              fieldType = "timestamptz";
+              // we also dont need to add a partition field, since it already exists.
+              addPartitionField = false;
+            }
```

Comment:
Does this mean the feature only lets you partition on fields of this type? Seems a bit limited